CONTRIBUTING.md

```diff
@@ -5,9 +5,14 @@ email, or any other method with the owners of this repository before making a ch
 
 Please note we have a code of conduct, please follow it in all your interactions with the project.
 
+## Commits messages
+
+This project follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) format for commit messages and implements it through the [pre-commit](https://pre-commit.com/) framework. Please install this as directed and enable it by running `pre-commit install`. 
+
+
 ## Pull Request Process
 
-1. Install [pre-commit](https://pre-commit.com/) framework and enable it running `pre-commit install`
+1. Install [pre-commit](https://pre-commit.com/) framework and enable it by running `pre-commit install`
 1. Ensure any install or build dependencies are removed before the end of the layer when doing a
    build.
```

Comment:
```suggestion
1. Ensure that you remove any dependencies before the end of the layer before you do a build.
```

Reply:
hmmm what are we doing here? what is the actionable step? how do we remove dependencies? what is "do a build"? hi! and help me understand please.

Reply:
Hi @noandrea, could you have a look at this when you have a moment? I don't quite understand the intent here. Might need some more detail about how to do this.

Reply:
not applicable to our context, I didn't catch it before when I reviewed the template